package.json

```diff
@@ -41,7 +41,7 @@
     "test:legacy:watch": "karma start --legacy --auto-watch=true --single-run=false",
     "test:update-snapshots": "karma start --update-snapshots",
     "test:prune-snapshots": "karma start --prune-snapshots",
-    "test:bs": "karma start karma.bs.config.js --legacy --coverage",
+    "test:bs": "karma start karma.bs.config.js --coverage",
```

Comment:
I also had to remove `--legacy` here since it is not compatible with new native ES modules